packages/closer/components/LessonDescription/index.tsx

```
@@ -17,10 +14,10 @@ const LessonDescription: FC<Props> = ({ lesson }) => {
 
   return (
     <section className="mb-6 flex flex-col gap-6">
-      {initialRenderComplete && (
+      {initialRenderComplete && fullText && (
         <p
           className="rich-text"
-          dangerouslySetInnerHTML={{ __html: lesson.description }}
+          dangerouslySetInnerHTML={{ __html: fullText }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)